Android/ChoreCenterApp/app/src/main/java/com/cse403chorecenter/chorecenterapp/UserLogin.java

```diff
@@ -61,6 +63,8 @@ protected void onStart() {
                 .requestIdToken("551695683870-tn6t4q27f5qpfe8jb61dt0jbr6qjf1fm.apps.googleusercontent.com")  // set up Google API console project to use here
                 .build();
         ACCOUNT_TYPE = intent.getStringExtra(MainActivity.EXTRA_MESSAGE);
+        ACCOUNT_DISPLAY_NAME = "Android";
```

Comment:
Is there a way to have the hard-coded strings as a resource in the string file?

Reply:
Yes we can always simply put those hard-coded strings in the resource string file. I only used hard-coded strings for behaviors that are not specified. For example, in this case, I am not sure if displaying "Android" as default is a good choice when the user somehow dropped network connection when entering the navigation page and maybe the request for user information would fail.